root/programs/C#/Frameworks/Infrastructure/Business/RichClient/Presentation/MyBaseControllerWin.cs

```diff
@@ -114,7 +115,23 @@ private void addControlEvent()
             ////    this, GetConfigParameter.GetConfigValue("FxPrefixOfToolStripMenuItem"),
             ////    new System.EventHandler(this.Button_Click), this.ControlHt);
             #endregion
+           
+            // コントロール検索＆イベントハンドラ設定
+            RcMyCmnFunction.GetCtrlAndSetClickEventHandler2(this, this.MyCreatePrefixAndEvtHndHt(), this.ControlHt);
 
+            #endregion
+        }
+
+        /// <summary>
+        /// コントロールのプレフィックスと
+        /// イベント ハンドラのディクショナリを生成
+        /// </summary>
+        /// <returns>
+        /// プレフィックスと
+        /// イベント ハンドラのディクショナリ
+        /// </returns>
+        protected Dictionary<string, object> MyCreatePrefixAndEvtHndHt()
```

Comment:
@daisukenishino 

Is *Ht* an abbreviation for *HashTable*?
If so, since the type of return value is *Dictionary*, use *Dic* instead.